docs/admin/configuration.rst

```diff
@@ -22,17 +22,17 @@ parameters using EdgeQL.  For example:
     CONFIGURE SYSTEM SET listen_addresses := {'127.0.0.1', '::1'};
 
 
-edgedb configure
-================
+edgedb config
```

Comment:
Oh, sorry, this is going to be reverted: https://github.com/edgedb/edgedb-cli/issues/376

Reply:
I've reverted that bit (and other similar mentions) back to `configure`.